packages/fxa-content-server/app/styles/modules/_branding.scss

```diff
@@ -36,6 +36,10 @@
   }
 }
 
+#main-content.send-sms::before {
+  content: initial;
+}
+
```

Comment:
You may have already checked with Alex/Ryan, but does he want the button to be this disabled blue instead of the grey shown in the Feature Doc? (I think the blue makes the most sense, just double checking this is what design wants)

<img width="400" alt="image" src="https://user-images.githubusercontent.com/13018240/76218252-e70ac500-61e1-11ea-868f-29f5d69a4fda.png">

![image](https://user-images.githubusercontent.com/13018240/76218353-1c171780-61e2-11ea-97e5-e9d2f3885d34.png)